src/main/kotlin/io/snyk/plugin/ui/SnykSettingsDialog.kt

```diff
@@ -288,6 +291,32 @@
             )
         )
 
+        val organizationContextHelpLabel = ContextHelpLabel.createWithLink(
+            null,
+            SnykBundle.message("snyk.settings.organization.tooltip.description"),
+            SnykBundle.message("snyk.settings.organization.tooltip.linkText")
+        ) {
+            BrowserUtil.browse(SnykBundle.message("snyk.settings.organization.tooltip.link"))
+        }
```

Comment:
shell we may be refactoring that class to useUIUtilsKt.baseGridConstraints and/or UIUtilsKt.baseGridConstraintsAnchorWest as we do now in all other UI components?...

Reply:
In my opinion if we want to refactor -> we should refactor the whole class. So I choose to keep PR scope small.